### PR TITLE
Payf 3456 doc update 2

### PR DESCRIPTION
**Pull request by jonhageman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1265,7 +1165,61 @@ <h4 id="example-get-all-patient-types">Example: Create Payment Response</h4>
...
+                <tbody>
+                    <tr>
+                        <td>id</td>
+                        <td>The Id for the payment intended to be canceled</td>
+                        <td>Guid</td>
+                    </tr>
+                </tbody>
+            </table>
+            <h4>Response Object</h4>
+            <p><a href="#payment-dto">PaymentDTO</a></p>
```
Comment (l-rodriguez) [inline index.html:1197]: change PaymentDTO for PaymentFullDTO

`<a href="#payment-full-dto">List of PaymentFullDTO</a>`

File: index.html [hunk 2]
```diff
@@ -1090,30 +1075,15 @@ <h4 id="example-get-all-patient-types">Request Parameters</h4>
...
+                        <td>Guid</td>
                     </tr>
                 </tbody>
             </table>
             <h4>Response Object</h4>
             <p><a href="#payment-full-dto">PaymentFullDTO</a></p>
             <h4 id="example-get-all-patient-types">Example: Get Payment By ID Request</h4>
             <pre class="center-column">
-GET https://payfac.nextech.com/api/pulbic/payments/3289
+GET https://payfac.nextech.com/api/public/payments/3289
```
[the remark this answers is not in the document]
Comment (jonhageman) [reply index.html]: done

File: index.html [hunk 3]
```diff
@@ -1090,30 +1075,15 @@ <h4 id="example-get-all-patient-types">Request Parameters</h4>
...
  -H 'partnerId: af43ba54;'
```
[the remark this answers is not in the document]
Comment (jonhageman) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (jonhageman) [reply index.html]: done

**Review by l-rodriguez — APPROVED** [review]
lgtm!

**Pull request merged**